### ISSUE TITLE
Sponge Detected When FTB installed, FORCE USE OF FTB COMMAND OVERRIDING

### ISSUE THREAD

**Issue by Tareku99** [FTBTeam/FTB-Utilities#1060]
Hello guys, been using FTBUtilities hard core. super cool and useful. I recently added sponge as well to run some plugins. I am not a mod savvy (still new to it) but I did see that there was a change in the code where command overriding would be disabled if sponge was detected. 
Now, i am trying to manage plugin permissions with FTB (from the discord replies, it can do it) but since now command overriding is disabled.. it isn't working anymore (or the ranks for that matter). 
I would like to request if there was a way to force command overriding when sponge is installed (maybe a new option in the config file?) or maybe a workaround :D

**Comment by KillingKanzaki** [comment]
Did you ever get a solution for this @Tareku99 ? I'm having the same issue

**Comment by Tareku99** [comment]
@KillingKanzaki nope. as great as sponge is .. i had to drop it.. too many issues with the amount of mods i was using.
but maybe the new forge will help you ?! test that one out

**Comment by KillingKanzaki** [comment]
Yo! I was able to fix it be uninstalling any plug inside that wasn't being used i.e chopchop and dependencies, and then disabled and renabled the force chat names settings in ftbutilities. Now got sponge and a differe tree capitation plugin working fine along rank names and ftb revelation + galacticraft

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
